packages/web-components/fast-components/src/default-palette.ts

```diff
@@ -96,101 +96,101 @@ export const neutralPalette: string[] = [
     "#151515",
     "#121212",
     "#101010",
-    "#000000"
```

Comment:
Somthing is going on with these trailling commas - I see some PRs removing them and some adding them.

Reply:
I'm guessing it's a prettier mismatch, chris can you try updating?